docs/management/advanced-options.asciidoc

```diff
@@ -335,11 +336,11 @@ changes.
 
 [horizontal]
 [[courier-batchsearches]]`courier:batchSearches`::
-**Deprecated in 7.6. Starting in 8.0, this setting will be optimized internally.**
```

Comment:
Same problem with this one.

Reply:
Ditto, these changes have been reverted.